codegen-compiler/src/main/kotlin/com/freeletics/khonshu/codegen/codegen/ActivityGenerator.kt

```diff
@@ -49,48 +46,18 @@ internal class ActivityGenerator(
                 compositionLocalProvider,
                 localActivityComponentProvider,
             )
-            .apply {
-                if (data.experimentalNavigation) {
-                    beginControlFlow("val useExperimentalNavigation = %M(component)", remember)
-                        .addStatement("component.useExperimentalNavigation")
-                        .endControlFlow()
-                        .beginControlFlow("if (useExperimentalNavigation)")
-                        .addCode(callNavHost(true))
-                        .nextControlFlow("else")
-                        .addCode(callNavHost(false))
-                        .endControlFlow()
-                } else {
-                    addCode(callNavHost(false))
-                }
-            }
-            .endControlFlow()
-            .endControlFlow()
-            .endControlFlow()
-            .build()
-    }
-
-    private fun callNavHost(experimental: Boolean): CodeBlock {
-        return CodeBlock.builder()
-            .apply {
-                if (experimental) {
-                    addStatement("%M(", experimentalNavHost)
-                } else {
-                    addStatement("%M(", androidxNavHost)
-                }
-            }
+            .addStatement("%M(", experimentalNavHost)
```

Comment:
```suggestion
            .addStatement("%M(", navHost)
```

What do you say about renaming this to `navHost` to prevent confusion?